src/ia-item-user-lists.ts

```diff
@@ -89,22 +89,50 @@ export class IaItemUserLists extends LitElement {
       // eslint-disable-next-line no-undef
       createEventListener as EventListener
     );
+
+    // Listens for `iaux-userlist-settings` errors from create-new-list
+    const listsSettingsErrorEventListener = (e: CustomEvent) => {
+      // eslint-disable-next-line no-console
+      console.log('userListError listener', e.detail);
+      this.dispatchEvent(
+        new CustomEvent('userListSettingsError', {
+          detail: { ...e.detail },
+        })
+      );
+    };
+    window.addEventListener(
```

Comment:
Question: shouldn't this event listener be removed when the component is disconnected?